xbmc/FileItem.cpp

```diff
@@ -490,8 +476,7 @@ void CFileItem::Reset()
   m_epgInfoTag.reset();
   delete m_pvrChannelInfoTag;
   m_pvrChannelInfoTag=NULL;
-  delete m_pvrRecordingInfoTag;
-  m_pvrRecordingInfoTag=NULL;
+  m_pvrRecordingInfoTag.reset();
```

Comment:
No I'm not, but not for everything that touches recordings. It makes perfect sense for methods that first created a FileItem copy, then passed it to GUI, which then had to find the correct recording in the list again based on the copy. But not so much for Reset(), which will very likely lead to issues, because nobody expects that calling Reset() on a fileitem clears out the contents of a recording in CPVRRecording...